terminal/tcell/cell_options.go

```diff
@@ -63,7 +63,6 @@ func cellOptsToStyle(opts *cell.Options, colorMode terminalapi.ColorMode) tcell.
 	fg := cellColor(colorToMode(opts.FgColor, colorMode))
 	bg := cellColor(colorToMode(opts.BgColor, colorMode))
 
-	// FIXME: tcell doesn't have a strikethrough style option until #254 is resolved.
-	st = st.Foreground(fg).Background(bg).Bold(opts.Bold).Italic(opts.Italic).Underline(opts.Underline)
+	st = st.Foreground(fg).Background(bg).Bold(opts.Bold).Italic(opts.Italic).Underline(opts.Underline).StrikeThrough(opts.Strikethrough).Reverse(opts.Inverse).Blink(opts.Blink)
```

Comment:
(nit, optional) This is starting to be a fairly long line to read. Consider breaking it, I think Go allows a line break on a period, something like:

```go
st = st.Foreground(fg).
    Background(bg).
    Bold(opts.Bold).
    ...
```